packages/fxa-auth-server/lib/payments/capability.ts

```diff
@@ -361,11 +394,12 @@ export class CapabilityService {
           );
         });
       }
-    } catch (error) {
-      this.log.error('subscriptions.getPlanEligibility', { error: error });
+    } catch (err) {
+      this.log.error('subscriptions.getPlanEligibility', { error: err });
       Sentry.captureException(error);
```

Comment:
Not necessary to change from error to err in this instance. But if you do, make sure to change the Sentry.caputreException() to use err as well.